packages/yoastseo/src/worker/AnalysisWebWorker.js

```diff
@@ -260,6 +262,17 @@ export default class AnalysisWebWorker {
 		this._relatedKeywordAssessor = this.createRelatedKeywordsAssessor();
 	}
 
+	/**
+	 * Sets the options to use for the Inclusive language analysis.
+	 *
+	 * @param {{infoLinks: {}}} options The options to use.
+	 *
+	 * @returns {void}
+	 */
+	setInclusiveLanguageOptions( options ) {
+		this._inclusiveLanguageOptions = options;
+	}
+
```

Comment:
@hansjovis When we added additional methods in the worker like `setCustomSEOAssessorClass`, we also bind it inside the constructor. But we don't do it to `setInclusiveLanguageOptions` here. Do you know if "binding" or not "binding" it to the scope will have no difference for `setInclusiveLanguageOptions`?

Reply:
Yes, binding it inside of the constructor would be the best option. To avoid any edge cases when calling the method. 

Could you implement that? 🙂 I don't have time at the moment.

Reply:
I'll do that! :)